### ISSUE TITLE
Steam deck setup.sh freeze

### ISSUE THREAD

**Issue by imayoda** [joshua-holmes/beammp-installer#3]
Hi there, 
I have a problem trying to execute this installer, it freezes just after
`fsync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely`

I have looked around to see if needed files are in place, proton experimental available, conf is populated and tried bash +x but with no luck.
We are in need for clues :) 
Thanks in advance

**Comment by MixerFLX** [comment]
Yeah, you aren't the only one facing that problem. From what I've seen, I'm pretty sure it's fsync in proton causing the error.
